From f92c1d300deffe02ad2f13df920dfc82781b7384 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Mon, 5 Jun 2017 10:44:08 +0800
Subject: [PATCH 1929/5242] MLK-15001-28 arm64: dts: fsl-imx8qm-lpddr2-arm2:
 Enable dpu1 and dpu2

commit  19cbdb5a667b1a0401b8422d2a6ddac16c25bc3e from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch enables dpu1 and dpu2.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index c361885..eb7ffb2 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -516,3 +516,11 @@
 &imx8_gpu_ss {
 	status = "okay";
 };
+
+&dpu1 {
+	status = "okay";
+};
+
+&dpu2 {
+	status = "okay";
+};
-- 
1.7.9.5

